src/main/java/org/folio/rest/impl/OaiPmhSetImpl.java

```diff
@@ -90,7 +92,7 @@ public void postOaiPmhSets(String lang, FolioSet entity, Map<String, String> oka
       Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     vertxContext.runOnContext(v -> {
       try {
-        logger.info("Post set with body: {0}", entity);
+        logger.info("Post set with body: {}", entityToJsonString(entity));
```

Comment:
```suggestion
        logger.info("Post set with body: {}", () -> entityToJsonString(entity));
```

Reply:
I tried the approach but the compiler swears on it.
![3](https://user-images.githubusercontent.com/60663044/111319598-d8fe4880-866e-11eb-9872-6ab50b62e7ec.png)
I will back to it at the bugfix ticket PR's